src/components/AccountNotifications.tsx

```diff
@@ -91,24 +91,26 @@ export const AccountNotifications: FC<IAccountNotifications> = (
         </div>
       )}
 
-      {showAccountNotifications && groupByRepository
-        ? Object.values(groupedNotifications).map((repoNotifications) => {
-            const repoSlug = repoNotifications[0].repository.full_name;
+      {showAccountNotifications
+        ? groupByRepository
```

Comment:
As a fix, this is fine, but as code to maintain: I find this hard to reason with. i.e. What do two chained `?` operators do? :)

Reply:
Do you find this easier to reason with?

```
{showAccountNotifications && (
  <>
    {groupByRepository
      ? Object.values(groupedNotifications).map((repoNotifications) => {
        const repoSlug = repoNotifications[0].repository.full_name;

        return (
          <RepositoryNotifications
			key={repoSlug}
			repoName={repoSlug}
			repoNotifications={repoNotifications}
		  />
		);
	  })
	: notifications.map((notification) => (
	  <NotificationRow
		key={notification.id}
		notification={notification}
	  />
	))}
  </>
)}
```

Reply:
https://github.com/gitify-app/gitify/pull/1306